security-insights-schema-1.0.0.yaml

```diff
@@ -178,10 +178,8 @@ properties:
                         type: string
                     package-url:
```

Comment:
Are you young to remove the two fields above as PURL encodes everything we need?